config/locales/pdfs/certificate.en.yml

```diff
@@ -2,41 +2,64 @@ en:
   waste_exemptions_engine:
     pdfs:
       certificate:
-        heading: Certificate of Registration
-        regulator:
-          heading: Regulation authority
-          name: Name
-          address:
-            label: Address
-            line1: National Customer Service Centre
-            line2: 99 Parkway Avenue
-            line3: Sheffield
-            line4: S9 4WF
-          telephone:
-            label: Telephone number
-            number: "03708 506506"
-        certify_message: The Environment Agency certify that the following information is entered in the public register which they maintain.
-        registration:
+        heading: Waste exemption registration %{registration_number}
+        confirmation:
+          heading: Waste exemption registration %{registration_number}
+          paragraph1: We confirm that the details you have entered, as shown below, are now on the exemptions register. Your registration should show on the public register within 5 days.
+          warning: This document is not a permit. Exemptions may have ceased or been revoked.
+          paragraph2: "To check whether these exemptions are still valid, check the company’s registration %{company_registration_number} on the public register of waste exemptions:"
+        business_responsibilities:
+          heading: Business responsibilities
+          paragraph1_html: >
+            The business or organisation responsible for carrying out the exempt waste operations agrees to:
+            <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-2">
```

Comment:
I'd prefer to break this list into individual lines of YAML and keep all the styling in the HTML template – this is more consistent with how we've done other lists the past (example: https://github.com/DEFRA/waste-exemptions-engine/blob/master/config/locales/mailers/confirmation_mailer.en.yml#L12-L15) We've generally tried to avoid including HTML in the YAML unless it's something really tiny like a `<strong>` tag.

Reply:
will do :)